app/views/organisations/not_live.html.erb

```diff
@@ -22,7 +22,7 @@
     <div class="inner-block">
       <div class="description">
         <%= govspeak_to_html @organisation.summary %>
-        <% unless @organisation.closed? %>
+        <% unless @organisation.closed? || @organisation.court_or_hmcts_tribunal? %>
```

Comment:
I don't love the more complicated conditional logic this adds to the template. I'd rather see this pushed into a presenter, but since there isn't one at the moment and that would involve a rather large piece of refactoring, it shouldn't stop this work being merged.